mono/mini/Makefile.am.in

```diff
@@ -111,6 +117,11 @@ boehm_static_libraries = libmini.la $(interp_libs_with_mini) $(dbg_libs_with_min
 boehm_binaries  = mono-boehm
 endif
 
+if ENABLE_NETCORE
+netcore_libraries = libcoreclr.la
```

Comment:
does the library need to be called libcoreclr or can we call it something like libcoreclrbridge too to avoid confusion?